src/api.ts

```diff
@@ -0,0 +1,105 @@
+export interface Paginated<Type> {
+  count: number;
+  readonly results: Type[];
+}
+
+export interface Contributor {
+  name: string;
+  orcid: string;
+  roles: string[];
+}
+
+export interface StudyForm {
+  studyName: string;
+  piName: string;
+  piEmail: string;
+  piOrcid: string;
+  linkOutWebpage: string[];
+  studyDate: Nullable<string>;
+  description: string;
+  notes?: string;
+  contributors: Contributor[];
+}
+
+// TODO: replace the `object` types with the actual types
```

Comment:
Thanks for including this `TODO` comment. Do you know where the actual type information would come from, if someone were to want to do this replacement? For example, is there a specific file (or directory) in another repository, that the authoritative type information is currently in?